scipy/sparse/_sputils.py

```diff
@@ -388,6 +390,27 @@ def is_pydata_spmatrix(m) -> bool:
     return base_cls is not None and isinstance(m, base_cls)
 
 
+def convert_pydata_sparse_to_scipy(
+    args: Iterable[Any], target_format: Optional[L["csc", "csr"]] = None
+) -> Iterable[Union[Any, "sp.spmatrix"]]:
+    """
+    Convert any pydata/sparse arrays to scipy sparse matrices,
+    pass through anything else.
+    """
+    result = []
+    for arg in args:
+        if is_pydata_spmatrix(arg):
+            arg = arg.to_scipy_sparse()
+            if target_format == "csc":
+                arg = arg.tocsc()
```

Comment:
You can use `arg.asformat(target_format)` to simplify this.

Reply:
Done!